src/api/app/views/webui/image_templates/index.html.haml

```diff
@@ -24,6 +26,12 @@
         - if project.packages.empty?
           %p
             There are currently no base image templates available for project <strong>#{title_or_name(project)}</strong>.
-  - if @projects.empty?
+  - if first
+    .templates_form
+      = form_tag({ controller: :package, :action => :branch, project: "placeholder", package: "placeholder"}, :method => :post, id: "appliance_form") do
+        = label_tag(:target_package, 'Name your appliance', class: "name_appliance")
```

Comment:
Please add some indications what is allowed for this. Maximum 200 characters, no blank, /, :, - or ; (see Package.valid_name)